samples/cellular/nrf_cloud_multi_service/README.rst

```diff
@@ -921,8 +921,8 @@ For example:
 
 .. _nrf_cloud_multi_service_building_wifi_scan:
 
-Building with nRF7002 EK Wi-Fi scanning support (for nRF9160 DK)
-================================================================
+Building with nRF7002 EK Wi-Fi scanning support (for nRF91 Series DK)
```

Comment:
Note above, @glarsennordic

Reply:
I'll spend some time trying out the various overlay combos and report back in a few days on what is supported and what isn't

Reply:
These overlays are definitely not working, I'll try to find out why in more detail tomorrow

Reply:
Thank you @glarsennordic, let's see what you find out tomorrow. If it is a simple change we can add it here, if not, or if it takes more time to figure out, we can take that in a separate PR.

Reply:
I spent some time today looking into it. I've isolated it to a problem with our GPIO expander, and technically it looks like the problem actually doesn't prevent the sample from working, but it does print some rather nasty errors. It will take some additional time for us to figure out a root cause, so go ahead and remove the references to NRF9161 + 7002EK for the time being

Reply:
I've removed the nRF9161DK + nRF7002EK support from this PR.

Reply:
@eivindj-nordic I received info in Teams on how to fix this for the 9161DK and the same instructions work for the 9160DK.

I opened a draft PR here: https://github.com/nrfconnect/sdk-nrf/pull/12222

but you are welcome to cherry pick the commit from it and use it here, in which case I'll close my PR.

Reply:
Thank you @plskeggs. I've cherry-picked your commit here with changes requested by @trantanen in #12222 .